hypothesis-python/tests/cover/test_explicit_examples.py

```diff
@@ -173,7 +173,12 @@ def test_prints_verbose_output_for_explicit_examples():
     def test_always_passes(x):
         pass
 
-    assert_falsifying_output(test_always_passes, "Trying explicit", x="NOT AN INTEGER")
+    assert_falsifying_output(
+        test_always_passes,
+        expected_exception=None,
+        example_type="Trying explicit",
+        x="NOT AN INTEGER",
+    )
```

Comment:
This one is weird because it actually expects the test to *not* raise an error. Our `assert_falsifying_output` didn't actually support this, but it ended up tricking itself by accepting the `AssertionError` that is raised when the test (correctly) doesn't raise anything.